test/test_cromwell_workflow.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+CROMWELL_USER=$1
+CROMWELL_PASSWORD=$2
+CROMWELL_URL=$3
+INPUTS_JSON=$4
+WDL_FILE=$5
+OPTIONS_FILE=$6
+DEPENDENCIES_JSON=$7
+
+echo "Starting workflow."
+WORKFLOW_HASH=$(cromwell-tools run \
```

Comment:
i need to look at this `cromwell tools`, seems super handy

Reply:
Check out [cromwell-manager](https://github.com/ambrosejcarr/cromwell-manager/blob/master/src/examples/basic_functionality.ipynb) as well.